SPECS/sm.spec

```diff
@@ -74,6 +74,7 @@ Patch1026: 0026-feat-LVHDSR-add-a-way-to-modify-config-of-LVMs-60.patch
 Patch1027: 0027-reflect-upstream-changes-in-our-tests.patch
 Patch1028: 0028-CA-398425-correctly-check-for-multiple-targets-in-iS.patch
 Patch1029: 0029-Synchronization-with-8.2-LINSTOR-before-a-stable-rel.patch
+Patch1030: 0030-linstor-Fix-to-add-sync-to-DD.patch
```

Comment:
Does this patch affect code which is common with other drivers, or is it code which only involves linstor?

Reply:
A few other drivers use this function, we will test the behavior with @Wescoeur before merging in sm.
It only make it so DD sync his block accesses before finishing.
Ensuring there is no inflight requests when dd has finished.
It should be a pretty minor behavior change.

Reply:
So it's something we might contribute upstream, too?

For what operations do drivers use dd?

Reply:
It's used to zero the footer of VHD specifically in the GC.
It is specifically an issue for a problem with the linstor driver.